src/Player.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Created by lbregula on 11/29/16.
+ */
+public class Player {
+private String name;
+private int money;
+private String token;
+
+public Player(String name, String token){
+    this.name = name;
+    this.money = 1500;
+    this.token = token;
+}
+
+public String getName() {return this.name;}
+
+public int getMoney() {return this.money;}
```

Comment:
While this is a stylistic critique lets not do single line functions.  It is generally uncommon to have code all on one line in java.